lib/open_project/custom_styles/color_themes.rb

```diff
@@ -61,7 +63,7 @@ module ColorThemes
         theme: "OpenProject Light",
         colors: {
           "primary-button-color" => PRIMER_PRIMARY_BUTTON_COLOR,
-          "accent-color" => "#175A8E",
+          "accent-color" => "#1A67A3",
```

Comment:
The values here are the same as `DEPRECATED_PRIMARY_COLOR`. Why should we use a deprectated value? But if that is correct, why not reference the constant?